.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+const { config } = require('@dhis2/cli-style')
+
+module.exports = {
+    extends: [config.eslintReact],
+    rules: {
+        'max-params': [
+            2,
+            {
+                max: 4,
+            },
+        ],
```

Comment:
Why did this need to be overridden here?

Reply:
Unfortunately some redux actions take 4 arguments and I preferred to keep the type of changes that would be required to fix this out of this PR.